netbox/extras/forms/model_forms.py

```diff
@@ -107,7 +107,7 @@ def clean_extra_choices(self):
         data = []
         for line in self.cleaned_data['extra_choices'].splitlines():
             try:
-                value, label = line.split(',', maxsplit=1)
+                value, label = line.split(':', maxsplit=1)
```

Comment:
We probably want to provide a means to escape colons in the choices value or label. We got away without this for commas (so far), but I expect using colons as delimiters will present an issue without this ability. Something like `re.split('(?<!\\\\):', line)` would allow us to split on colons not preceded by a backslash.